src/test/javascript/spec/service/orion/orion-assessment.service.spec.ts

```diff
@@ -111,48 +107,53 @@ describe('OrionAssessmentService', () => {
 
         testConversion(mockReader as any);
 
-        expect(alertErrorStub).to.have.been.calledOnceWithExactly('artemisApp.assessmentDashboard.orion.downloadFailed');
+        expect(alertErrorStub).toHaveBeenCalledTimes(1);
+        expect(alertErrorStub).toHaveBeenCalledWith('artemisApp.assessmentDashboard.orion.downloadFailed');
     });
 
     /**
      * Helper to test the conversion with the fileReader
      * @param mockReader mock reader to test
      */
     function testConversion(mockReader: FileReader) {
-        const isCloningSpy = spy(orionConnectorService, 'isCloning');
-        const exportSubmissionStub = stub(programmingAssessmentExportService, 'exportReposByParticipations');
-        const lockAndGetStub = stub(programmingSubmissionService, 'lockAndGetProgrammingSubmissionParticipation');
-        const readerStub = stub(window, 'FileReader');
+        const isCloningSpy = jest.spyOn(orionConnectorService, 'isCloning');
+        const exportSubmissionStub = jest.spyOn(programmingAssessmentExportService, 'exportReposByParticipations');
+        const lockAndGetStub = jest.spyOn(programmingSubmissionService, 'lockAndGetProgrammingSubmissionParticipation');
+        const readerStub = jest.spyOn(window, 'FileReader');
 
         // first it loads the submission
-        lockAndGetStub.returns(of(programmingSubmission));
+        lockAndGetStub.mockReturnValue(of(programmingSubmission));
         // then the exported file
         const response = new HttpResponse({ body: new Blob(['Stuff', 'in blob']), status: 200 });
-        exportSubmissionStub.returns(of(response));
+        exportSubmissionStub.mockReturnValue(of(response));
 
-        readerStub.returns(mockReader);
+        readerStub.mockReturnValue(mockReader);
 
         orionAssessmentService.downloadSubmissionInOrion(16, programmingSubmission, 0, false);
 
-        expect(isCloningSpy).to.have.been.calledOnceWithExactly(true);
-        expect(lockAndGetStub).to.have.been.calledOnceWith(11, 0);
+        expect(isCloningSpy).toHaveBeenCalledTimes(1);
+        expect(isCloningSpy).toHaveBeenCalledWith(true);
+        expect(lockAndGetStub).toHaveBeenCalledTimes(1);
+        expect(lockAndGetStub).toHaveBeenCalledWith(11, 0);
+        expect(exportSubmissionStub).toHaveBeenCalledTimes(1);
         // ignore RepositoryExportOptions
-        expect(exportSubmissionStub).to.have.been.calledOnceWith(16, [1]);
-        expect(readerStub).to.have.been.calledOnce;
+        expect(exportSubmissionStub).toHaveBeenCalledWith(16, [1], expect.anything());
```

Comment:
Why expect "anything" here?

Reply:
Because the last parameter is the repository export options, which are just hardcoded. I see no use in copying them into the test, because that would be just asserting true = true.

Reply:
Then please document this decision in a short comment above the expect statement. If that should be the "// ignore RepositoryExportOptions", this is not clear enough. Explicitly state that you refer to the `expect.anything()`. Basically like you did here, thanks.